api/graphql/server.js

```diff
@@ -17,7 +18,7 @@ async function startApolloServer(typeDefs, resolvers) {
       // TODO: Add auth middleware
     },
     csrfPrevention: true,
-    mocks: true,
+    mocks: {DateTime: () => new Date()},
```

Comment:
Add a sample link and image mock as well!!

Reply:
Reference: [Link](https://www.apollographql.com/docs/apollo-server/testing/mocking/#customizing-mocks)

Reply:
@abhibhaw 
do you mean I should create an image and link type or do you want me to add a sample link and image for location and image of Event type respectively

Reply:
Leave it!!